compiler/rustc_feature/src/active.rs

```diff
@@ -485,9 +485,6 @@ declare_features! (
     /// Allows `async || body` closures.
     (active, async_closure, "1.37.0", Some(62290), None),
 
-    /// Allows `[x; N]` where `x` is a constant (RFC 2203).
-    (active, const_in_array_repeat_expressions, "1.37.0", Some(49147), None),
```

Comment:
I think you need to put it into the https://github.com/rust-lang/rust/blob/e199123b4775efbe57f34c8765ba403a4d053b00/compiler/rustc_feature/src/removed.rs#L45 list of removed features